test/sample-data.json

```diff
@@ -245,4 +245,4 @@
       }
     }
   }
-}
+}
```

Comment:
Please, stop breaking the last newline.

Reply:
I honestly don't know why the newline keeps trying to get removed.  It might be the JSON editor I'm using (which is out of date anyways).  :/